### PR TITLE
update Grafana dashboard

### PR DESCRIPTION
**Pull request by ebattat**
Update Grafana dashboards for Func and Perf environment

1. Save the latest json in the repository
2. Update PostgreSQL runtime to 2 minutes for Pod, kata and VM due so suspect for inconsistency results. We suspect that 1 minutes is not enough time for getting a reliable results, start with 2 and increase if in case of inconsistency.

### CODE REVIEW

File: HOW_TO.md [hunk 1]
```diff
@@ -223,10 +223,10 @@ any template .yaml files.
       6. Save & test
 2. Open grafana dashboard benchmark-runner-report:
    1. Open grafana
-   2. Create(+) -> import -> paste [grafana/benchmark-runner-report.json](grafana/benchmark-runner-report.json) -> Load
+   2. Create(+) -> import -> paste [grafana/benchmark-runner-report.json](grafana/func/benchmark-runner-report.json) -> Load
```
Comment (RobertKrawitz) [inline HOW_TO.md:226]: Need to fix the link, not just the text.

File: HOW_TO.md [hunk 2]
```diff
@@ -223,10 +223,10 @@ any template .yaml files.
...
    3. Create panel from scratch or duplicate existing on (stressng/uperf)
    4. Configure the workload related metrics
-   5. Save dashboard -> share -> Export -> view json -> Copy to clipboard -> override existing one [benchmark-runner-report.json](grafana/benchmark-runner-report.json)
+   5. Save dashboard -> share -> Export -> view json -> Copy to clipboard -> override existing one [benchmark-runner-report.json](grafana/func/benchmark-runner-report.json)
```
Comment (RobertKrawitz) [inline HOW_TO.md:229]: Fix the link

File: README.md [hunk 3]
```diff
@@ -119,9 +119,9 @@ SAVE RUN ARTIFACTS LOCAL:
 ## Grafana dashboards
 
 There are 3 grafana dashboards templates:
-1. [benchmark-runner-ci-status-report.json](grafana/benchmark-runner-ci-status-report.json)
+1. [benchmark-runner-ci-status-report.json](grafana/func/benchmark-runner-ci-status-report.json)
```
Comment (RobertKrawitz) [inline README.md:122]: Fix the link

File: README.md [hunk 4]
```diff
@@ -119,9 +119,9 @@ SAVE RUN ARTIFACTS LOCAL:
...
    ![](media/benchmark-runner-ci-status.png)
-2. [benchmark-runner-report.json](grafana/benchmark-runner-report.json)
+2. [benchmark-runner-report.json](grafana/func/benchmark-runner-report.json)
```
Comment (RobertKrawitz) [inline README.md:124]: Fix the link

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]

File: HOW_TO.md (hunk 1 above)
Comment (ebattat) [reply HOW_TO.md]: Fixed

File: HOW_TO.md (hunk 2 above)
Comment (ebattat) [reply HOW_TO.md]: Fixed

File: README.md (hunk 3 above)
Comment (ebattat) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (ebattat) [reply README.md]: Fixed

File: HOW_TO.md [hunk 5]
```diff
@@ -223,10 +223,10 @@ any template .yaml files.
       6. Save & test
 2. Open grafana dashboard benchmark-runner-report:
    1. Open grafana
-   2. Create(+) -> import -> paste [grafana/benchmark-runner-report.json](grafana/benchmark-runner-report.json) -> Load
+   2. Create(+) -> import -> paste [grafana/func/benchmark-runner-report.json](grafana/func/benchmark-runner-report.json) -> Load
    3. Create panel from scratch or duplicate existing on (stressng/uperf)
    4. Configure the workload related metrics
-   5. Save dashboard -> share -> Export -> view json -> Copy to clipboard -> override existing one [benchmark-runner-report.json](grafana/benchmark-runner-report.json)
+   5. Save dashboard -> share -> Export -> view json -> Copy to clipboard -> override existing one [grafana/func/bbenchmark-runner-report.json](grafana/func/benchmark-runner-report.json)
```
Comment (RobertKrawitz) [inline HOW_TO.md:229]: Typo (bbenchmark => benchmark)

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
One typo to be fixed.

**Review by RobertKrawitz — APPROVED** [review]

**Pull request merged**